### PR TITLE
[chore] remove_duplicates from receiver tab

### PR DESCRIPTION
**Pull request by haslinghuis**
Fix to solve sonar cloud complaint about duplicated logic in receiver tab.

### CODE REVIEW

File: src/js/tabs/receiver.js
Comment (chmelevskij) [inline src/js/tabs/receiver.js:866]: This was so much easier to read though 🤔

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by limonspb — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by ctzsnooze — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
Would a stub 'wrapper' function be more readable, and less repetitive?

**Comment by haslinghuis** [comment]
@chmelevskij example please 😁

**Comment by blckmn** [comment]
> @chmelevskij example please 😁

Something like this?

`        function func(delta, channel, rate) {
            return delta * this.rateCurve.rcCommandRawToDegreesPerSecond(
                channel,
                rate.roll_rate,
                rate.rc_rate,
                rate.rc_expo,
                rate.superexpo,
                rate.deadband,
                rate.roll_rate_limit
            );
        }
        
        const roll = func(delta, FC.RC.channels[0], this.currentRates);
        const pitch = func(delta, FC.RC.channels[1], this.currentRates);
        const yaw = func(delta, FC.RC.channels[2], this.currentRates);`

**Comment by haslinghuis** [comment]
Thanks for the example but only `this.currentRates.superexpo` and `this.currentRates.deadband` are repeated, other values are different

**Pull request merged**
